src/authAgents/SessionAuthAgent.js

```diff
@@ -9,12 +9,18 @@ define(["structures/CAPIError", "storages/LocalStorage"], function (CAPIError, L
      *
      * @class SessionAuthAgent
      * @constructor
-     * @param credentials {Object} object literal containg credentials for the REST service access
-     * @param credentials.login {String} user login
-     * @param credentials.password {String} user password
+     * @param authInfo {Object} object literal containg the credentials (`login`
```

Comment:
Maybe it would worthwhile to prepare some examples for params. For instance, how should the authInfo object look like or where should it be generated?